recipes/mppp/all/patches/2.0.0-0002-remove-fmt-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3a5400a..7907741 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -406,7 +406,7 @@ endif()
+ # to print 128-bit integers.
+ set(_MPPP_MIN_FMT_VERSION "6.2")
+ if(MPPP_WITH_FMT)
+-    find_package(fmt ${_MPPP_MIN_FMT_VERSION} REQUIRED CONFIG)
```

Comment:
is this patch is still needed? the minimum version listed there is 6.2 - if CMake fails because the version is newer than this, you may have some luck with the following, without the need of a patch:

```
deps.set_property("fmt",  "cmake_config_version_compat", "AnyNewerVersion")
```

Reply:
@jcar87 
Thank you for your review.
Because there are compilation errors without patches, I add cmake_config_version_compat definition.
It is more easy way. 
Thanks!